docs/docs/recipes/manage-users/README.md

```diff
@@ -3,3 +3,9 @@ sidebar_position: 6
 ---
 
 # 🧑‍🚀 Manage users
+
+## Manage users on the Admin Console page
+
+## Manage users by Logto Core management API
+
+## Manage user data on the storage
```

Comment:
maybe we don't need this. this page can be just a summary of the section

Reply:
OK. Will rewrite in normal content with links like below instead of these headings in next PR.

<img width="778" alt="image" src="https://user-images.githubusercontent.com/10594507/174965210-a6f8de59-8416-41f5-9156-b6129ca0d715.png">